lib/galaxy/datatypes/proteomics.py

```diff
@@ -218,6 +650,29 @@ class UniProtXML(ProteomicsXml):
     root = "uniprot"
 
 
+class XquestXML(ProteomicsXml):
+    file_ext = "xquest.xml"
+    blurb = "XQuest XML file"
+    root = "xquest_results"
+
+
+class XquestSpecXML(ProteomicsXml):
+    """spec.xml"""
+    file_ext = "spec.xml"
+    blurb = 'xquest_spectra'
+    root = "xquest_spectra"
+
+
+class QCML(ProteomicsXml):
+    """qcml
+    https://github.com/OpenMS/OpenMS/blob/develop/share/OpenMS/SCHEMAS/mzQCML_0_0_5.xsd
+    https://github.com/OpenMS/OpenMS/blob/develop/share/OpenMS/SCHEMAS/qcML_0.0.7.xsd
```

Comment:
should this be a link to a specific commit ...?

Reply:
OK. Will do - even if there is versioning in the filename :)